vsts/pipelines/templates/_integrationJobTemplate.yml

```diff
@@ -31,6 +31,10 @@ parameters:
       - 60
       - dynamic
       - runtime
+  - name: golangVersions
+    type: object
+    default:
+      - dynamic
```

Comment:
same here.

Reply:
Sounds good to me, I've made the change. Thanks!